superduperdb/backends/mongodb/data_backend.py

```diff
@@ -9,14 +9,45 @@
 from superduperdb.backends.base.data_backend import BaseDataBackend
 from superduperdb.backends.ibis.field_types import FieldType
 from superduperdb.backends.mongodb.artifacts import MongoArtifactStore
+from superduperdb.backends.base.metadata import MetaDataStoreProxy
 from superduperdb.backends.mongodb.metadata import MongoMetaDataStore
 from superduperdb.base.enums import DBType
 from superduperdb.components.datatype import DataType
 from superduperdb.misc.colors import Colors
+from superduperdb.backends.mongodb.utils import get_avaliable_conn
 
 from .query import MongoQuery
 
 
+def reconnect(self):
+    # Reconnect to database.
+    conn, _ = _MongoConnectionBuilder.build(self.uri, self.flavour)
+    self.conn = conn
+    self._db = self.conn[self.name]
+
+class _MongoConnectionBuilder:
```

Comment:
`???ConnectionBuilder` overkill.

Reply:
I can remove the class no problem.
I thought in future we might have complex logic hence added a class to accommodate helper functions.